packaging/linux/flatpak/install-mouse

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cp /app/share/sunshine/udev/rules.d/85-sunshine.rules $HOME/.config/sunshine
+flatpak-spawn --host pkexec sh -c "sudo usermod -a -G input $USER && cp $HOME/.config/sunshine/85-sunshine.rules /etc/udev/rules.d"
```

Comment:
I think we don't need `cp` and `rm` commands. Not tested though.

Would this work?
```bash
flatpak-spawn --host pkexec sh -c "sudo usermod -a -G input $USER && cat "/app/share/sunshine/udev/rules.d/85-sunshine.rules" | sudo tee /etc/udev/rules.d/85-sunshine.rules
```